src/main/scala/Column.scala

```diff
@@ -1134,6 +1156,35 @@ case class Column private (
     }
   }
 
+  def validateValue(value: String): Array[ErrorWithoutContext] = {
+    var errors = Array[ErrorWithoutContext]()
+    if (minInclusive.isDefined && value.toDouble < minInclusive.get) {
```

Comment:
What happens if the number is too big or too small to be represented by a `double`? Don't we need to do something here where we parse the number and then cope with `BigDecimals`?

Also won't this function break when the string value can't be parsed to a number?